src/python/pants/backend/python/subsystems/python_setup.py

```diff
@@ -62,13 +62,19 @@ def register_options(cls, register):
     register('--artifact-cache-dir', advanced=True, default=None, metavar='<dir>',
              help='The parent directory for the python artifact cache. '
                   'If unspecified, a standard path under the workdir is used.')
+    register('--fixed-interpreter', advanced=True, default=None, metavar='<interpreter-binary>',
```

Comment:
this should probably be an interpreter search path that's checked against the constraints vs a single interpreter.. in order to support e.g. 2 + 3. we've learned this the hard way with `PEX_PYTHON` (vs e.g. `PEX_PYTHONS`).

Reply:
So I'd pass those paths into `interpreter_cache.setup()`? I can do that